lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -1501,8 +1501,9 @@ def test_calling_conventions(self):
             ax.voxels(x, y)
         # x, y, z are positional only - this passes them on as attributes of
         # Poly3DCollection
-        with pytest.raises(AttributeError):
+        with pytest.raises(AttributeError) as exec_info:
             ax.voxels(filled=filled, x=x, y=y, z=z)
+        assert exec_info.value.name == 'x'
```

Comment:
This was the only test I found that covers the changed code.  Should I add something new or is this enough?

Reply:
Take or leave my optional suggestions. I don’t think this needs additional tests.

Reply:
Take or leave my optional suggestions. I don’t think this needs additional tests.